.github/workflows/ci.yml

```diff
@@ -46,3 +46,23 @@ jobs:
 
             - name: ✨ Run code style check
               run: composer cs
+
+    phpstan:
+      runs-on: ubuntu-latest
+      name: Code style
+      steps:
+        - name: 📤 Checkout project
+          uses: actions/checkout@v2
+
+        - name: 🐘 Install PHP
+          uses: shivammathur/setup-php@v2
+          with:
+            php-version: 7.1
+            extensions: intl, tidy, bcmath, zip
```

Comment:
do we need all these extensions for `search-v3`?

Reply:
I can try without them, I thought they were needed for PHPStan?

Reply:
`zip` seems especially weird 🤔 
Maybe try without them and re-add those that are necessary? And maybe we should also add those that are required to `composer.json` then?

Reply:
I will re-add the PHP extensions one by one.

Reply:
Seems like none of them are needed. Makes sense for this package I think. Should we double-check other repositories as well?

Reply:
I can do that yes. Maybe I take this list?

- https://github.com/cultuurnet/udb3-silex
- https://github.com/cultuurnet/calendar-summary-v3
- https://github.com/cultuurnet/search-v3
- https://github.com/cultuurnet/udb3-search-service
- https://github.com/cultuurnet/jwt-provider
- https://github.com/cultuurnet/udb3-api-guard